docs/src/usage/std.md

```diff
@@ -78,6 +78,24 @@ Example:
       type: any
 ```
 
+#### "must_use"
+This field is used for checking if the return value of a function is used.
+
+- `true` - The return value of this function must be used.
+- `false` - The default. The return value of this function does not need to be used.
```

Comment:
Default option should come first.

Reply:
Resolved in: https://github.com/Kampfkarren/selene/pull/583/commits/78b1523b8d906fbc608a977b9c17d6d03bc1d72e